netket/optimizer/qgt/qgt_jacobian_dense_logic.py

```diff
@@ -151,21 +151,21 @@ def _rescale(centered_oks):
         mpi.mpi_sum_jax(
             jnp.sum((centered_oks * centered_oks.conj()).real, axis=0, keepdims=True)
         )[0]
-        ** 0.5
-    )
+        + offset
+    ) ** 0.5
     centered_oks = jnp.divide(centered_oks, scale)
     scale = jnp.squeeze(scale, axis=0)
     return centered_oks, scale
 
 
-@partial(jax.jit, static_argnames=("apply_fun", "mode", "rescale_shift", "chunk_size"))
+@partial(jax.jit, static_argnames=("apply_fun", "mode", "chunk_size"))
```

Comment:
Shouldn't `offset` be static now?

Reply:
No. It jits the same way if it's any `float`, and `None` generates a new jit by type.

Reply:
Though it makes little difference, it would usually be the same number throughout...